rhoas-client/src/main/resources/application.properties

```diff
@@ -0,0 +1,21 @@
+event-bridge.rhoas.mgmt-api.host=https://api.openshift.com
+event-bridge.rhoas.instance-api.host=http://localhost
+event-bridge.rhoas.sso.red-hat.auth-server-url=https://sso.redhat.com/auth/realms/redhat-external
+event-bridge.rhoas.sso.red-hat.refresh-token=dummy-refresh-token
+event-bridge.rhoas.sso.mas.auth-server-url=https://identity.api.openshift.com/auth/realms/rhoas
+event-bridge.rhoas.sso.mas.client-id=dummy-client-id
+event-bridge.rhoas.sso.mas.client-secret=dummy-client-id
+
+### OIDC Clients for RHOAS
+quarkus.oidc-client.client-enabled=false
+# Red Hat SSO (refresh token read from "event-bridge.rhoas.sso.red-hat.refresh-token" property)
+quarkus.oidc-client.red-hat-sso.auth-server-url=${event-bridge.rhoas.sso.red-hat.auth-server-url}
+quarkus.oidc-client.red-hat-sso.client-id=cloud-services
+quarkus.oidc-client.red-hat-sso.credentials.secret=secret
+quarkus.oidc-client.red-hat-sso.grant.type=refresh
+# MAS SSO
+quarkus.oidc-client.mas-sso.auth-server-url=${event-bridge.rhoas.sso.mas.auth-server-url}
+quarkus.oidc-client.mas-sso.client-id=${event-bridge.rhoas.sso.mas.client-id}
+quarkus.oidc-client.mas-sso.credentials.secret=${event-bridge.rhoas.sso.mas.client-secret}
+quarkus.oidc-client.mas-sso.grant.type=client
+quarkus.oidc-client.mas-sso.scopes=email
```

Comment:
Are these fixed values?

Reply:
Yes (based on the current MAS SSO auth flow)